src/component/description/cases/observing/Styles.css

```diff
@@ -0,0 +1,18 @@
+.MuiFab-primary-5 {
```

Comment:
You shouldn't be overriding this here. Why don't you use the styles object in the `Fab`. You can set margin right and bottom there.

Example:

```
const styles = theme => ({
  ...
  fab: {
    position: 'absolute',
    bottom: theme.spacing.unit * 2,
    right: theme.spacing.unit * 2,
  },
  ...
});
```

Please read the docs here:

https://material-ui.com/demos/buttons/